Configurations/unix-Makefile.tmpl

```diff
@@ -844,6 +844,24 @@ tags TAGS: FORCE
 	-ctags -R .
 	-etags `find . -name '*.[ch]' -o -name '*.pm'`
 
+# Find the sources for the FIPS module ('providers/fips')
+# Only take the C and H files
+# Shave off $(SRCDIR) before processing
+fips-checksums: FORCE
+	if which unifdef > /dev/null; then \
+	    $(SRCDIR)/util/getsource.pl providers/fips \
+	        | cut -f2 -d: | grep '\.[ch]$$' \
+	        | sed -e "s|$(SRCDIR)/||" \
+	        | ( cd $(SRCDIR); xargs ./util/fips-checksums.sh \
+	                                > ./util/fips-sources.checksums ) \
+	    && \
+	    openssl sha256 -hex \
+	            < $(SRCDIR)/util/fips-sources.checksums \
+	            > $(SRCDIR)/util/fips.checksum; \
+	else \
+	    echo >&2 "WARNING: unifdef not in your \$$PATH, FIPS checksums not calculated"; \
```

Comment:
This should be ERROR not warning, no?

Reply:
Not initially.  I'm not about to punish everyone who runs `make update` and hasn't installed `unifdef` yet...